src/Avalonia.Themes.Default/Window.xaml

```diff
@@ -8,6 +8,7 @@
       <Panel>
         <Border Name="PART_TransparencyFallback" IsHitTestVisible="False" />
         <Border Background="{TemplateBinding Background}" IsHitTestVisible="False" />
+        <Panel Background="Transparent" Margin="{TemplateBinding WindowDecorationMargin}" />
```

Comment:
Why not just enable hit testing on the border as before? This panel AFAICS will be in the same place as the border.

Reply:
Because the native titlebar relies on the window not intercepting hits to detect if it was pressed or dragged I.e to move window. So you want an area that isn't hit testable to fill the background, and an area that is but obeys the margin provided via the binding to shift out the way of the native titlebar.

Reply:
Ah yeah sorry, I missed the `Margin`.